benchmarks/modules/benchmark_conv.py

```
@@ -0,0 +1,99 @@
+                    weight=weight,
+                    bias=bias,
+                    activation='swish',
+                    seq_idx=prepare_sequence_ids(cu_seqlens).to(torch.int32).unsqueeze(0),
+                ),
+                'b d t -> b t d'
+            ),
+            quantiles=quantiles
+        )
+    elif provider.startswith('causal_conv1d_fwdbwd'):
+        results = triton.testing.do_bench(
+            lambda: causal_conv1d(x, weight, bias, activation='swish', cu_seqlens=cu_seqlens, residual=True).backward(x),
```

Pylint (3.3.7)

[convention] Line too long (121/100)

(C0301)